### PR TITLE
[EN/ZH] Tournaments-CJB Ro16

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 1]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
 
 ## Mappool
 
-Coming Soon...
+### Round of 16
+
+[Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA)  
+[Google Drive](https://drive.google.com/file/d/1ZNekV5Lh6pw3qgl9BuWEs6CN6dSpOl8N)
+
+- [Loos - Koi Yumi Zakura \[Stage 4: Story\]](https://osu.ppy.sh/beatmapsets/791641/#mania/1660726)
```
Comment (WalterToro) [inline wiki/Tournaments/CJB/2018/4K/en.md]: I would point that these maps are part of the FreeMod bracket, and i would write like this:

- Freemod
  - Map one
  - Map two
  - etc.
- Tiebreaker
  - Map

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 2]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
+- [DJ TOTTO - Crystalia \[CS' Hyper\]](https://osu.ppy.sh/beatmapsets/782749/#mania/1643587)
+- [The Quick Brown Fox - The Big Black \[Challenge\]](https://osu.ppy.sh/beatmapsets/708264/#mania/1497402)
+- [cYsmix - Breeze \[Hard\]](https://osu.ppy.sh/beatmapsets/639924/#mania/1357114)
+- [FictionJunction YUUKA - Yakusoku \[Pinky Swear\]](https://osu.ppy.sh/beatmapsets/594912/#mania/1258189)
+- [goreshit - burn this moment into the retina of my eye \[Retina\]](https://osu.ppy.sh/beatmapsets/685427/#mania/1450885)
+- [xi - .357 Magnum \[Hyper\]](https://osu.ppy.sh/beatmapsets/155749/#mania/564542)
+- [senya - Zetsubou no Fuchi \[540's Hard\]](https://osu.ppy.sh/beatmapsets/709521/#mania/1500034)
+- [ak+q - Excelsia \[Stage 5: Freestyle\]](https://osu.ppy.sh/beatmapsets/791699/#mania/1660811)
+
+TieBreaker:
```
Comment (WalterToro) [inline wiki/Tournaments/CJB/2018/4K/en.md]: TieBreaker > Tiebreaker

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 3]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
 
 - [Livestream](https://live.bilibili.com/5333)
 - [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp)
+- [challonge](https://challonge.com/zh_CN/cjb4)
+
+## Match schedule
+
+### Round of 16
+
+Saturday，21. July 2018
```
Comment (WalterToro) [inline wiki/Tournaments/CJB/2018/4K/en.md]: I would use a header level 4 on these dates > `#### Saturday，21. July 2018`

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 4]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
+| 15:00 | Snowkyou | ![][flag_CN] | ![][flag_CN] | IA daisuki | 3 |
+| 16:00 | Zhou_Kai | ![][flag_TW] | ![][flag_CN] | Shindou Ayane | 7 |
+| 20:00 | crazybrother | ![][flag_CN] | ![][flag_CN] | williammak | 8 |
 
 ## Mappool
 
-Coming Soon...
+### Round of 16
+
+[Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA)
```
Comment (WalterToro) [inline wiki/Tournaments/CJB/2018/4K/en.md]: I would use a single line mappacks like this (also add the size of the mappack replacing the XX.X MB part):
**Download the mappack here! (XX.X MB) - [Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA) - [Google Drive](https://drive.google.com/file/d/1ZNekV5Lh6pw3qgl9BuWEs6CN6dSpOl8N)**

`**Download the mappack here! (XX.X MB) - [Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA) - [Google Drive](https://drive.google.com/file/d/1ZNekV5Lh6pw3qgl9BuWEs6CN6dSpOl8N)**`

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 5]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
 
 - [Livestream](https://live.bilibili.com/5333)
 - [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp)
+- [challonge](https://challonge.com/zh_CN/cjb4)
```
Comment (WalterToro) [inline wiki/Tournaments/CJB/2018/4K/en.md]: challonge > Challonge

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 6]
```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
+
+## Match schedule
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: `schedule` -> `Schedule`

File: wiki/Tournaments/CJB/2018/4K/en.md (hunk 3 above)
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Wrong comma. Fix other occurrences.

**Pull request merged**